### ISSUE TITLE
Using multiple channel?

### ISSUE THREAD

**Issue by rikkuporta** [dennybritz/cnn-text-classification-tf#107]
Hi, the original papers mentions suing multiple channels - could you point me into the right direction how I could implement this?

**Comment by rikkuporta** [comment]
I solved it by aranging the input as [batch, channel, sequencelength] and then feeding it to the placeholder as `self.X = tf.placeholder(tf.int32, [None, num_channels, seq_length], name='X')`
My other channel are something similiar to the same word in another "language" so i created a word embedding for each channel in a for-loop ` embedings.append(tf.nn.embedding_lookup(W, self.X[:,i,:]))` and then stacking it on a new last axis `self.word_embedding_expanded = tf.stack(embedings, axis=3)`

Finallz I changed the filter shape to refelct the filter as `filter_shape = [filter_size, embedding_size, num_channels, num_filters]`
